packages/grafana-ui/src/components/Table/TableNG/Cells/PillCell.tsx

```diff
@@ -93,11 +104,14 @@ function getPillColor(value: string, field: Field, theme: GrafanaTheme2): string
 }
 
 export const getStyles = (theme: GrafanaTheme2) => ({
+  wrapper: css({
+    marginBlockStart: theme.spacing(-0.25),
+  }),
```

Comment:
We should be able to kill this wrapper div with the paradigm introduced in https://github.com/grafana/grafana/pull/108048, where we can create a custom class within TableNG for Cell to take.